web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -328,7 +328,13 @@ const PulseReportPage = () => {
     });
     if (res.error) return;
 
-    const pulses = res.payload.data;
+    // Get the pulses and filter out invalid data.
+    const pulses = res.payload.data.filter((pulse) => {
+      return pulse.internalScore > 0 && pulse.internalScore <= 5 &&
+             (pulse.externalScore == null ||
+              (pulse.externalScore > 0 && pulse.externalScore <= 5));
+    });
```

Comment:
Do we have bad data somewhere?

Reply:
There is nothing that restricts externalScore from being any integer value.  When I was testing, just for fun, I decided to send values outside of that range and it did not deal well.